packages/expo/Expo.podspec

```diff
@@ -42,8 +61,45 @@ Pod::Spec.new do |s|
     $expo_warned_about_missing_autolinking = true
   end
 
-  s.source_files = 'ios/**/*.{h,m,swift}'
+  header_search_paths = [
+    '"$(PODS_ROOT)/Headers/Private/React-Core"', # as React-RCTAppDelegate.podspec to access JSCExecutorFactory.h
+    '"$(PODS_ROOT)/DoubleConversion"',
```

Comment:
I can add the `DoubleConversion` from '"${PODS_CONFIGURATION_BUILD_DIR}/DoubleConversion/DoubleConversion.framework/Headers"' because it has a different folder structure.

Basically, you can import it as: `<DoubleConversion/double-conversion.h>` or `<double-conversion/double-conversion.h>`. I need the second version to import `folly/dynamic.h`.

Reply:
I would prefer not to add this call. It appears to add much more, and including more unnecessary dependencies may complicate the build graph.